include/wx/fontutil.h

```diff
@@ -161,8 +161,16 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
 
     void Free();
 
+    // Font Names in macOS and iOS:
+    // while GetFaceName from now on returns the unique and never localized PostScript name of the font
+    // there are other types of names that are useful
+    
+    // returns the font family name eg Helvetica Neue
     wxString GetFamilyName() const;
+    // returns the font style name eg Condensed Bold
     wxString GetStyleName() const;
+    // returns the font display name (for UI), might be localized, eg Helvetica Neue Condensed Bold
```

Comment:
Sorry, I don't understand the example: "Helvetica Neue" was given as the example of non-localised PS name for `GetFamilyName()`, yet here it's used as an example of a localized name. Am I missing something or is one of the examples incorrect?

Reply:
Sorry, the comment lines are above the methods, I've just copied them from the source. I've added the proper PostScript name to the initial paragraph and added whitespace to make things hopefully clearer

Reply:
Stefan, when you say that you've added the proper PS name, where is it? Have you not updated the PR yet or am I just misunderstanding you?

Reply:
I just added it in the initial paragraph of this conversion